### PR TITLE
Add PowerMate.app v1.2.4_0

### PR DESCRIPTION
**Pull request by tmcinerney**
Add the Griffin PowerMate USB and Bluetooth controller application.

### CODE REVIEW

File: Casks/powermate.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'powermate' do
+  version '1.2.4_0'
+  sha256 '6fd755961bb5a486cb3ab9d47217f7877d4bb021134f32a3c32ceede3fd3cf3d'
+
+  url "http://support.griffintechnology.com/sites/default/files/PowerMate_v#{version}.zip"
+  name 'Griffin Powermate'
+  homepage 'http://support.griffintechnology.com/support/powermate/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/powermate.rb]: Please read [this](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#license-stanza-details) and pick the appropriate one.

Comment (vitorgalvao) [inline Casks/powermate.rb]: There should be a `tags :vendor => 'Griffin'` line under this one.

Comment (tmcinerney) [inline Casks/powermate.rb]: Thanks @vitorgalvao! Just fixed that and pushed an update.

Comment (vitorgalvao) [inline Casks/powermate.rb]: Thank you.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 02b73e6333cbd4fb0be30016399ef43bbe920fd8 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**